src/Namotion.Reflection/Context/ContextualAccessorInfo.cs

```diff
@@ -43,7 +43,7 @@ public abstract class ContextualAccessorInfo : ContextualMemberInfo
         /// <returns>The attribute or null.</returns>
         public T? GetContextAttribute<T>()
         {
-            return ContextAttributes.OfType<T>().SingleOrDefault();
```

Comment:
SingleOrDefault<T>?

Reply:
now using `GetSingleOrDefault`, different name for easier to recognize as non-LINQ